actioncable/lib/action_cable/connection/client_socket.rb

```diff
@@ -7,6 +7,19 @@ module Connection
     #
     # Copyright (c) 2010-2015 James Coglan
     class ClientSocket # :nodoc:
+      CLOSE_CODES = {
```

Comment:
Heavily based on this https://github.com/faye/websocket-driver-ruby/blob/master/lib/websocket/driver/hybi.rb#L37-L47 This is the list of non reserved codes that we can use to close the web socket from the server. I included the whole list to validate the close code here https://github.com/rails/rails/pull/29390/files#diff-d0938ccf792c521c268ce7daef16fea1R105